### PR TITLE
COMMERCE-12688 Fixed wish list add to cart button

### PR DESCRIPTION
**Pull request by alessiorendina**
https://liferay.atlassian.net/browse/COMMERCE-12688

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-wish-list-web/src/main/resources/META-INF/resources/wish_list_item_action.jsp [hunk 1]
```diff
@@ -22,6 +22,7 @@ CProduct cProduct = commerceWishListItem.getCProduct();
 	<c:when test="<%= cpInstance != null %>">
 		<commerce-ui:add-to-cart
 			alignment="left"
+			CPCatalogEntry="<%= commerceWishListDisplayContext.getCPCatalogEntry(cProduct.getPublishedCPDefinitionId()) %>"
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-wish-list-web/src/main/resources/META-INF/resources/wish_list_item_action.jsp:25]: looking at the code this is the right fix, but why cpCatalogEntry is not mandatory if without this nothing makes sense?
I think we should make it mandatory

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-wish-list-web/src/main/resources/META-INF/resources/wish_list_item_action.jsp (hunk 1 above)
Comment (alessiorendina) [reply modules/apps/commerce/commerce-wish-list-web/src/main/resources/META-INF/resources/wish_list_item_action.jsp:25]: Yes, let' open a story for this

Comment (alessiorendina) [reply modules/apps/commerce/commerce-wish-list-web/src/main/resources/META-INF/resources/wish_list_item_action.jsp:25]: https://liferay.atlassian.net/browse/COMMERCE-12737

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
The issues are not related to the changes, PR can be forward!

@victoroscuro

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**